.github/workflows/continuous-deploy.yml

```diff
@@ -24,21 +24,19 @@ jobs:
         with:
           node-version: "18.14.1"
 
-      - name: Yarn Install
-        run: yarn install
```

Comment:
I don't think we should've deleted this step but rather keep it and add a new build step:
```
- name: Yarn Install
   run: yarn install

- name: Yarn Build
   run: yarn build
```

This way the `static` folder is updated with the latest compiled files which should be published to cloudflare pages

Reply:
I agree. Perhaps you can incorporate this change now?